src/Containers/App/App.tsx

```diff
@@ -1,22 +1,16 @@
-import React, { Component } from "react";
-import Routes from "./AppRoutes";
-import "./App.css";
-import Header from "../../Components/Header";
+import React, {Component} from 'react';
+import { HashRouter, Route, Switch } from 'react-router-dom';
+import DefaultLayout from '../../Components/DefaultLayout/DefaultLayout';
```

Comment:
should have an index.ts file within each folder to export necessary modules as rules of "ducks"